.travis.yml

```diff
@@ -12,9 +12,11 @@ jobs:
   include:
   - env: CMD="make validate"
   - env: CMD="make deploy"
-         CLUSTERS_ARGS="--globalnet"
-         DEPLOY_ARGS="${CLUSTERS_ARGS} --deploytool helm"
+         CLUSTERS_ARGS="--globalnet --timeout 1m"
+         DEPLOY_ARGS="--globalnet --deploytool helm --timeout 2m"
```

Comment:
You are passing "globalnet=true" to DEPLOY_ARGS as well. Is this intentional?

Reply:
If you notice it was `${CLUSTERS_ARGS}` before so it was already getting it, but since i don't want to have "conflicting" timeouts I opted to specify it explicitly